test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/DatabaseHandlerTests.cs

```diff
@@ -374,6 +378,90 @@ public async Task VerifyDatabaseScopedConfigurationsTest()
             }
         }
 
+        /// <summary>
+        /// Adding, modifying and deleting database files
+        /// </summary>
+        /// <returns></returns>
+        [Test]
+        public async Task VerifyDatabaseFilesTest()
+        {
+            // setup, drop database if exists.
+            var connectionResult = await LiveConnectionHelper.InitLiveConnectionInfoAsync("master", serverType: TestServerType.OnPrem);
+            using (SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connectionResult.ConnectionInfo))
+            {
+                var server = new Server(new ServerConnection(sqlConn));
+
+                var testDatabase = ObjectManagementTestUtils.GetTestDatabaseInfo();
+                var objUrn = ObjectManagementTestUtils.GetDatabaseURN(testDatabase.Name);
+                await ObjectManagementTestUtils.DropObject(connectionResult.ConnectionInfo.OwnerUri, objUrn);
```

Comment:
Why would the DB exist? We should be making a new DB for each test run, so that should never happen (and doing this could hide other problems if we are accidently sharing test objects)